sql/core/src/test/scala/org/apache/spark/sql/internal/ExecutorSideSQLConfSuite.scala

```diff
@@ -40,16 +40,24 @@ class ExecutorSideSQLConfSuite extends SparkFunSuite with SQLTestUtils {
     spark = null
   }
 
+  private def withStaticSQLConf(pairs: (String, String)*)(f: => Unit): Unit = {
```

Comment:
ah sorry I was wrong. Static conf is no different from normal conf, it's just immutable during runtime. Maybe just call this method `withSQLConf`?

Reply:
will check

Reply:
In case of `withSQLConf`, I got an exception;
```
- SPARK-24727 CODEGEN_CACHE_SIZE is correctly referenced at the executor side *** FAILED ***
  org.apache.spark.sql.AnalysisException: Cannot modify the value of a static config: spark.sql.codegen.cacheSize;
  at org.apache.spark.sql.catalyst.plans.PlanTestBase$$anonfun$withSQLConf$1.apply(PlanTest.scala:172)
```

Reply:
ah sorry for the confusion about "call", I mean rename this method...